Assets/MRTK/SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -215,12 +215,12 @@ public Vector3 SliderTrackDirection
         public SliderEvent OnHoverExited = new SliderEvent();
         #endregion
 
-        #region Private Members
-        private float startSliderValue;
-        private Vector3 startPointerPosition;
-        private Vector3 startSliderPosition;
-        private IMixedRealityPointer activePointer;
-        private Vector3 sliderThumbOffset = Vector3.zero;
+        #region Protected Members
+        protected float startSliderValue;
```

Comment:
Please make sure to add [summary comments](https://microsoft.github.io/MixedRealityToolkit-Unity/version/releases/2.3.0/Documentation/Contributing/CodingGuidelines.html#function--method-summary-headers) to these new protected methods, as they will now be part of the documentation.